apps/collection/acceptance/pages/nominated-person.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+let I;
+
+module.exports = {
+
+  _init() {
+    I = require('so-acceptance/steps.js')();
```

Comment:
Do we ever use any other files from `so-acceptance`? I think we should consider setting that as `main` in the module.